setup.py

```diff
@@ -21,20 +21,20 @@
 ]
```

Comment:
Above here in requirements can you also allow django 1.11, it's what we're using

Reply:
Sure